src/DbSchemaManager.php

```diff
@@ -191,51 +194,51 @@ public function ensureTables(): void
      */
     public function ensureNoTables(): void
     {
-        if ($this->hasTable($this->itemsChildrenTable)) {
+        if ($this->itemsChildrenTable !== null && $this->hasTable($this->itemsChildrenTable)) {
             $this->dropTable($this->itemsChildrenTable);
         }
 
-        if ($this->hasTable($this->assignmentsTable)) {
+        if ($this->assignmentsTable !== null && $this->hasTable($this->assignmentsTable)) {
             $this->dropTable($this->assignmentsTable);
         }
 
-        if ($this->hasTable($this->itemsTable)) {
+        if ($this->itemsTable !== null && $this->hasTable($this->itemsTable)) {
             $this->dropTable($this->itemsTable);
         }
     }
 
     /**
      * Gets name of the table for storing RBAC items (roles and permissions).
      *
-     * @return string Table name
+     * @return string|null Table name.
      *
      * @see $itemsTable
      */
-    public function getItemsTable(): string
+    public function getItemsTable(): ?string
```

Comment:
To this change need adapt `RbacCycleInit`. May be add tests for `RbacCycleInit` (sugggest make this in separate PR) ?

Reply:
OK. Let's do it separately - #63.